samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <ctype.h>
+#include <gpio.h>
+#include <stdio.h>
+#include <net/lwm2m.h>
+#include <net/bsdlib.h>
+#include <settings/settings.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(app_lwm2m_client, CONFIG_APP_LOG_LEVEL);
+
+#include <at_cmd.h>
+#include <lte_lc.h>
+
+#if defined(CONFIG_LWM2M_DTLS_SUPPORT)
+#include "nrf_inbuilt_key.h"
+#endif
+
+#include "ui.h"
+#include "lwm2m_client.h"
+#include "settings.h"
+
+#if !defined(CONFIG_LTE_LINK_CONTROL)
+#error "Missing CONFIG_LTE_LINK_CONTROL"
+#endif
+
+BUILD_ASSERT_MSG(sizeof(CONFIG_APP_LWM2M_SERVER) > 1,
```

Comment:
Won't this check prevent the CI from building the sample successfully?

Reply:
Added a fix to pass CI (hopefully)